code/modules/client/preferences_toggles.dm

```diff
@@ -264,6 +264,7 @@
 		"<a href='?src=\ref[src];action=proccall;procpath=/client/proc/toggle_help_intent_safety'>Toggle Help Intent Safety</a><br>",
 		"<a href='?src=\ref[src];action=proccall;procpath=/client/proc/toggle_auto_eject'>Toggle Guns Auto-Ejecting Magazines</a><br>",
 		"<a href='?src=\ref[src];action=proccall;procpath=/client/proc/toggle_auto_eject_to_hand'>Toggle Guns Auto-Ejecting Magazines to Your Hands</a><br>",
+		"<a href='?src=\ref[src];action=proccall;procpath=/client/proc/toggle_ammo_display_type'>Semi-auto ammo counter</a><br>",
```

Comment:
These toggles normally also exist in https://github.com/cmss13-devs/cmss13/blob/5da327a71f6da77ba0d82e71d4aba80f9a42166a/code/modules/client/preferences.dm#L621